backend/mutations.py

```diff
@@ -301,7 +313,33 @@ def mutate(self, info, id):
         return DeleteUserMutation(success=success)
 
 class ArtworkMetricsInput(graphene.InputObjectType):
+    total_visits = graphene.Int()       # number of visits to artwork
+    total_ratings = graphene.Int()      # number of ratings submitted to artwork
+
+class DiscoverArtworkMutation(graphene.Mutation):
+    
+    class Arguments:
+        user_id = graphene.ID(required=True)
+        artwork_id = graphene.ID(required=True)
+
     total_visits = graphene.Int()
+    works_visited = graphene.Int()
+
+    def mutate(self, info, user_id=None, artwork_id=None):
+        artwork = UpdateArtworkMutation.getArtwork(artwork_id)
+        artwork.metrics.total_visits += 1
+        artwork.save()
+        total_visits = artwork.metrics.total_visits
+
+        user = UpdateUserMutation.getUser(user_id)
```

Comment:
ohhh now I see the beauty of those static functions. Makes sense with the class name too.